src/main/scala/io/github/spark_redshift_community/spark/redshift/DefaultSource.scala

```diff
@@ -49,7 +54,16 @@ class DefaultSource(
       sqlContext: SQLContext,
       parameters: Map[String, String]): BaseRelation = {
     val params = Parameters.mergeParameters(parameters)
-    redshift.RedshiftRelation(jdbcWrapper, s3ClientFactory, params, None)(sqlContext)
+    val jdbcOptions = new JDBCOptions(CaseInsensitiveMap(parameters))
```

Comment:
Get this flag from the spark conf, not the jdbc options. Like this https://github.com/ActionIQ/flame/blob/10cb9caf35f5ba7f488fd7d8a5ef268de4366812/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcRelationProvider.scala#L39-L41

Reply:
Also very similar code below, can have this method call the one below

Reply:
Hmm, sqlContext.conf is not accessible from here. It is a private[spark] thing
https://github.com/ActionIQ/flame/blob/87bcff99a6fba000bf67a129582035514ede39a7/core/src/main/scala/org/apache/spark/SparkContext.scala#L222

Reply:
look a few lines below

Reply:
Asking a clarifying questions:
Do you mean this? 
```
// calling the below createRelation
 override def createRelation(
      sqlContext: SQLContext,
      parameters: Map[String, String]): BaseRelation = {
       createRelation(sqlContext, parameters, None)
}
// changed this function to 
override def createRelation(
      sqlContext: SQLContext,
      parameters: Map[String, String],
      schema: Option[StructType])
```

But we don't have that power to change spark code, so we can't rewrite function [here] easily (https://github.com/ActionIQ/flame/blob/beed5e20af0a3935ef42beb3431c8630599bf27f/sql/core/src/main/scala/org/apache/spark/sql/sources/interfaces.scala#L107).

Reply:
i meant sparkContext.getConf

Reply:
What is the biggest consideration for not using a jdbcOption here to control extra pushdown?

I find it hard to add a unit test since we can't change the value of `sparkContext.getConf` in runtime. Maybe we don't need unit test in redshift code since we have those in jdbc already?